src/main.rs

```diff
@@ -118,7 +120,8 @@ fn main() {
     let subscriber = tracing_fmt::FmtSubscriber::builder()
         .with_ansi(color)
         .with_filter(tracing_fmt::filter::EnvFilter::from(levels.as_str()))
-        .finish();
+        .finish()
+        .with(Limit::default());
```

Comment:
I am currently in the process of updating all of this right now, so I would hold off on any specific tracing PRs for the moment.

Reply:
Oh, ok, I can adapt this PR later